### PR TITLE
Adding test open command

### PR DESCRIPTION
**Pull request by tiagoemsi**
# Description
This PR adds test to the open command.

# How to test?
The test must be approved by the CI automatically.

# Expected behavior
No behavior is changed.

### CODE REVIEW

File: internal/commands/open_test.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	testName             string
+	expectError          bool
+	errorMessage         string
+	ctx                  context.Context
+	mockLogger           log.Logger
+	mockClient           bot.Client
+	mockRepository       model.Repository
+	mockFilestorage      filestorage.Driver
+	triggerID            string
+	mockDialogSubmission bot.DialogSubmission
```
Comment (GLOFonseca) [inline internal/commands/open_test.go:28]: Not a request.
I like to break these fields in fields related to the test, fields that are parameters of the function, mocks. Just an idea.

File: internal/commands/open_test.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	repositoryMock.On("InsertIncident", mock.AnythingOfType("*model.Incident")).Return(int64(1), nil)
+	repositoryMock.On("AddPostMortemUrl", mock.AnythingOfType("string"), mock.AnythingOfType("string")).Return(nil)
+	filestorageMock.On("CreatePostMortemDocument", f.ctx, mock.AnythingOfType("string")).Return(string(""), nil)
+}
+
+func TestOpenIncidentDialog(t *testing.T) {
+	var f openCommandFixture
+	t.Run("Dialog created properly", func(t *testing.T) {
+		f.setup(t)
+		err := commands.OpenStartIncidentDialog(f.mockClient, f.triggerID)
```
Comment (GLOFonseca) [inline internal/commands/open_test.go:63]: This shows that maybe our code is wrong. You a sending an empty string and no error returns.

File: internal/commands/open_test.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+					SeverityLevel:       "2",
+					Product:             "A",
+					IncidentCommander:   "UYGFQB9C0",
+					IncidentDescription: "Incident Resolved!",
+				},
+			},
+		},
+		{
+			testName:    "When the form has no data",
+			expectError: false,
```
Comment (GLOFonseca) [inline internal/commands/open_test.go:93]: This also shows that our code accepts anything to open an incident.

Comment (tiagoemsi) [reply internal/commands/open_test.go:93]: Yes, we will need to improve the code

File: internal/commands/open_test.go (hunk 2 above)
Comment (tiagoemsi) [reply internal/commands/open_test.go:63]: I realized this when building the test :/

File: internal/commands/open_test.go (hunk 1 above)
Comment (tiagoemsi) [reply internal/commands/open_test.go:28]: Nice! I can improve it in v2

**Pull request merged**